ios/Sources/HomeFeature/HomeFeature.swift

```diff
@@ -0,0 +1,25 @@
+import ComposableArchitecture
+
+public struct HomeState: Equatable {
+    // TODO: Replace to real models
+    public var contents: [String]
```

Comment:
ditto

Reply:
This is possible to reference from other state, so I think this property should be public accessor.

Reply:
Please fix this one as well ref: https://github.com/DroidKaigi/conference-app-2021/pull/457/files?file-filters%5B%5D=.strings&file-filters%5B%5D=.swift#r644213196